tss-esapi/src/constants/command_code.rs

```diff
@@ -150,3 +154,67 @@ impl From<CommandCode> for TPM2_CC {
         command_code.to_u32().unwrap()
     }
 }
+
+impl Marshall for CommandCode {
+    const BUFFER_SIZE: usize = std::mem::size_of::<TPM2_CC>();
+
+    /// Produce a marshalled [TPM2_CC]
+    fn marshall(&self) -> Result<Vec<u8>> {
+        let mut buffer = vec![0; Self::BUFFER_SIZE];
+        let mut offset = 0;
```

Comment:
Should not marshall then behave the same way?

Reply:
By this you mean having a `marshall_offset` function as well?

Reply:
Exactly or maybe that was something that was intensionally left out?

Reply:
That's very true - I implemented this as I needed the unmarshall in particular, but it makes sense to make it "symmetrical" :) I'll add that as a commit on top of these two, stay tuned

Reply:
Added